clusterNode.py

```diff
@@ -130,19 +130,19 @@ def setup_vm(self, iso_or_image_path: str) -> host.Result:
         else:
             network = "bridge=virbr0"
         cmd = f"""
-        virt-install
-            --connect qemu:///system
-            -n {self.config.name}
-            -r {self.config.ram}
-            --cpu host
-            --vcpus {self.config.cpu}
-            --os-variant={self.config.os_variant}
-            --import
-            --network {network},mac={self.config.mac}
-            --events on_reboot=restart
-            {cdrom_line}
-            --disk path={self.config.image_path}
-            {append}
+        virt-install                                    \\
+            --connect qemu:///system                    \\
+            -n {self.config.name}                       \\
+            -r {self.config.ram}                        \\
+            --cpu host                                  \\
+            --vcpus {self.config.cpu}                   \\
+            --os-variant={self.config.os_variant}       \\
+            --import                                    \\
+            --network {network},mac={self.config.mac}   \\
+            --events on_reboot=restart                  \\
+            {cdrom_line}                                \\
+            --disk path={self.config.image_path}        \\
+            {append}                                    \\
```

Comment:
No. On the contrary.

On current `main` you cannot load a shell script, because `Host._run_remote()` mangles new lines.

If you write a shell script, and you wrap the lines (for one command), you will very much need those trailing backslashes.

Reply:
I don't like that we have to add `\\` in the scripts.

Reply:
This we need to fix before we consider merging this PR.